ophidian/circuit/Netlist.cpp

```diff
@@ -11,7 +11,7 @@ Netlist::Netlist() :
 {
 }
 
-Netlist::Netlist(Netlist &&nl) :
+Netlist::Netlist(const Netlist &&nl) :
```

Comment:
I'd suggest you to create a method called swap and use the[ copy-and-swap idiom](http://stackoverflow.com/questions/3279543/what-is-the-copy-and-swap-idiom). 
This way, you will create robust implementations for copy constructor, copy assignment, and move constructor.